api/cmd/main.go

```diff
@@ -62,7 +61,7 @@ func main() {
 
 	s3ws.Filter(logging.FilterFactory())
 	s3ws.Filter(context.FilterFactory())
-	s3ws.Filter(signer.FilterFactory())
+	//s3ws.Filter(signer.FilterFactory())
```

Comment:
commenting this line was a part of testing we do in our local environment, this should not be committed